Voycar.Api.Web.Tests.Integration/Common/Exceptions.cs

```diff
@@ -0,0 +1,24 @@
+namespace Voycar.Api.Web.Tests.Integration.Common;
+
+/// <summary>
+/// Serves as the base class for exceptions indicating missing items in the database, which are required
+/// for program execution.
+/// </summary>
+public abstract class ElementNotInDbException : Exception
+{
+    protected ElementNotInDbException() : base() { }
+    protected ElementNotInDbException(string message) : base(message) { }
+    protected ElementNotInDbException(string message, Exception inner) : base(message, inner) { }
+
+}
+
+/// <summary>
+/// The exception that is thrown when a specific role element is expected to be in the
```

Comment:
```suggestion
/// To be thrown when a specific role element is expected to be in the
```

Reply:
Ich habe versucht mich da an die .NET convention zu halten. Alle Exceptions in `System` sind scheinbar so kommentiert.

Reply:
Okay, dann kann das so bleiben. Find ich komisch.